ui-modules/blueprint-composer/app/components/dsl-viewer/dsl-viewer.template.html

```diff
@@ -46,6 +48,10 @@
             <dsl-viewer dsl="argument"></dsl-viewer>
         </li>
     </ol>
-</span>
+  </span>
 
-<span ng-if="isLiteralDsl(dsl)">{{dsl.name}}</span>
\ No newline at end of file
+  <span ng-switch-default>
+    {{ getMode(dsl) }}: {{ dsl }}
```

Comment:
the space here is very small, adding the mode is a waste of valuable real estate. We already have an icon to symbolise what type this is, adding the mode is an unnecessary repetition. I think it should be dropped.

You could however add that has a hover title of tooltip

Reply:
in this default case we do _not_ have an icon so i think this is the right way to do it (but i don't think we have any default cases anymore so it's moot)

Reply:
Fair enough